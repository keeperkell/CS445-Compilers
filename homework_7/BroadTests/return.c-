int null()
 begin 
    return 107;
 end 

int cat()
 begin 
    return null();
 end 

main()
 begin 
    int x;
    899;
    x <- cat();
    output(x);
 end 

