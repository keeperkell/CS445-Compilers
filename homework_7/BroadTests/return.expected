Loading file: BroadTests/return.tm
107
Bye.
